src/System.Collections.Concurrent/tests/System.Collections.Concurrent.Tests.builds

```diff
@@ -2,16 +2,16 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Collections.Concurrent.Tests.csproj"/>
     <Project Include="System.Collections.Concurrent.Tests.csproj">
-      <TargetGroup>netstandard1.7</TargetGroup>
-      <TestTFMs>netcoreapp1.1</TestTFMs>
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcoreapp1.0</TestTFMs>
     </Project>
+    <Project Include="System.Collections.Concurrent.Tests.csproj" />
```

Comment:
This won't be super straight forward to do, the reason why they got out of order like this is because I used a tool to parse and edit all of these files, but it edits them in place, so this one for example use to be the `netstandard1.7` and `netcoreapp1.1` build, but it is now the default since those are the defaults now. Since it doesn't really add any value I believe if we watne dto clean this up we could do it later, but I can totally do it now if you feel strongly about it.